lib/profiles.js

```diff
@@ -5,9 +5,9 @@ import FetchRequest from "./fetch-request";
  * Loads the schemas for all object types, with each key of the object being the @type for each
  * schema.
  * @param {object} session Authentication session object
- * @returns {Promise<object>} Promise that resolves to the /profiles object
+ * @returns {Promise<DataProviderObject | ErrorObject>} Promise that resolves to the /profiles object
  */
 export default async function getProfiles(dataProviderUrl) {
   const request = new FetchRequest();
-  return request.getObjectByUrl(`${dataProviderUrl}/profiles`, null);
+  return (await request.getObjectByUrl(`${dataProviderUrl}/profiles`)).union();
```

Comment:
Should this be `optional()` since the previous default is null, and session-context.js doesn’t really do anything with an error.

Reply:
oh yes, you're right